Applications/Analyze/test/Tug_of_War_Setup_Analyze.xml

```diff
@@ -44,7 +44,13 @@
 				</ForceReporter>
 				<OutputReporter name="OutputReporter">
 					<!--The names of Outputs to be reported. To select specific Component Outputs, provide its path name. For example, 'slider/tx/value' is an Output for the value of a Coordinate 'tx' belonging to the Joint 'slider'.-->
-					<output_paths>forceset/LeftMuscle/fiber_length forceset/LeftMuscle/tendon_force com_position forceset/LeftMuscle/active_force_length_multiplier jointset/ground_block/reaction_on_child</output_paths>
+					<output_paths>
+						forceset/LeftMuscle|fiber_length
+						forceset/LeftMuscle|tendon_force
+						|com_position
```

Comment:
Is the leading vertical bar necessary here? Looks strange and seems inconsistent with "forceset/..." above which has no prefix. IMHO a more intuitive interface would be to prefix with a keyword in this case (e.g., "model|com_position") or just drop the vertical bar if there's no path.